frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/BasePullBranchAction.java

```diff
@@ -147,6 +146,9 @@ public void run(ProgressIndicator indicator) {
       @Override
       public void onSuccess() {
         VcsNotifier.getInstance(project).notifySuccess("Fetch of refspec ${refspec} succeeded");
+        // For some reason, the call to `BaseGitMacheteGraphTable::queueRepositoryUpdateAndModelRefresh` within `actionPerformed`
+        // does not refresh the graph table after the pulls. Therefore, the following solution is to be used.
```

Comment:
Hmmm weird... did you research on what specific stage was the update stopped? As in, did it e.g. manage to queue a `GitMacheteRepositoryUpdateTask`?

Reply:
oh, it works now :rofl: It is possible that I did smth wrong before